src/features/game/expansion/components/UpcomingExpansionModal.tsx

```diff
@@ -42,6 +42,23 @@ export const UpcomingExpansionModal: React.FC<Props> = ({
   onClose,
   onExpand,
 }) => {
+  //We cannot expand if there is no next expansion
+  if (gameState.expansionRequirements === undefined) {
+    return (
+      <div className="flex flex-col items-center">
+        <span className="mb-2">No expansion available</span>
```

Comment:
@adamhannigan It should never really be true. It's just there for safety in case somehow that button does get clicked when it's not supposed to be visible.